CONTRIBUTING.md

```diff
@@ -11,9 +11,11 @@ take a moment to read through them.
 
 ### GitHub issues
 
-Team RabbitMQ uses GitHub issues for _specific actionable items_ engineers can work on. This assumes the following:
+Team RabbitMQ uses GitHub issues for _specific actionable items_ that
```

Comment:
I'm not a huge fan of the "Team Foobar" wording. To me, it sounds like a French trying to speak English, mixing French and English rules to contruct a sentence :-) Is it valid English?

Also, the rest of the document is using "the RabbitMQ team" which looks more correct to me.

Reply:
I will get it!

Reply:
@dumbbell FWIW I believe the UK national team at the 2012 London Olympics were referred to as "Team GB" everywhere (in the media, official athlete kit and merchandise). We are not an olympic team, of course, and it could be a Britishism :)